app/templates/courses/teacher-class-view.jade

```diff
@@ -101,19 +113,27 @@ block content
         .completeness-info.col-md-4
           h4.m-b-2
             &nbsp;
-          if state.get('earliestIncompleteLevel')
-            div.small-details
+          div.small-details
+            if state.get('earliestIncompleteLevel')
+              span(data-i18n='teacher.earliest_incomplete')
+              span :
+              +longLevelName(state.get('earliestIncompleteLevel'))
+              +inlineUserList(state.get('earliestIncompleteLevel').users)
+            else
               span(data-i18n='teacher.earliest_incomplete')
               span :
-            +longLevelName(state.get('earliestIncompleteLevel'))
-            +inlineUserList(state.get('earliestIncompleteLevel').users)
+              div(data-i18n='common.loading')
 
-          if state.get('latestCompleteLevel')
-            div.small-details.m-t-3
+          div.small-details.m-t-3
+            if state.get('latestCompleteLevel')
+              span(data-i18n='teacher.latest_complete')
+              span :
+              +longLevelName(state.get('latestCompleteLevel'))
+              +inlineUserList(state.get('latestCompleteLevel').users)
```

Comment:
How do these two work when the classroom is empty or no user has played a level? This logic looks like it'll always say "loading".